meta-lmp-bsp/conf/machine/include/lmp-machine-custom.inc

```diff
@@ -261,6 +261,37 @@ WKS_FILE_apalis-imx8_sota = "sdimage-imx8-sota.wks.in"
 PREFERRED_PROVIDER_u-boot-default-script_apalis-imx8 = "u-boot-base-scr"
 PREFERRED_PROVIDER_u-boot-default-script_sota_apalis-imx8 = "u-boot-ostree-scr-fit"
 
+# iMX8MQ
+UBOOT_SIGN_ENABLE_mx8mq = "1"
+PREFERRED_PROVIDER_virtual/bootloader_sota_mx8mq ?= "u-boot-fio"
+PREFERRED_PROVIDER_u-boot_sota_mx8mq ?= "u-boot-fio"
+PREFERRED_VERSION_u-boot-fio_mx8mq ?= "2020.04"
+PREFERRED_PROVIDER_u-boot-fw-utils_mx8mq = "libubootenv"
+PREFERRED_RPROVIDER_u-boot-fw-utils_mx8mq = "libubootenv"
+SOTA_CLIENT_FEATURES_append_mx8mq = " ubootenv"
+UBOOT_DTB_LOADADDRESS_mx8mq = "0x43000000"
+# TODO: fixup hard-coded dtb entry here
+IMAGE_BOOT_FILES_mx8mq = "imx-boot boot.scr uEnv.txt Image imx8mq-evk.dtb"
```

Comment:
Here you can probably just use '${@make_dtb_boot_files(d)}' for the device tree, as the list of supported device trees are already specified by KERNEL_DEVICETREE.